src/atoms/forms/checkbox-input.tsx

```diff
@@ -32,12 +33,10 @@ const CheckboxInput: React.FC<CheckboxInputProperties> = (
     const { checked, disabled, label, onChange } = props;
 
     let className = ELEMENT_CLASS;
-    if (disabled) {
-        className += " -disabled";
-    }
+    let isDisabled = disabled ? CheckboxDisabledClass : "";
```

Comment:
Suggestion: Recommend renaming from `isDisabled` to `isDisabledClassName` or something along those lines. While I realize we are in typescript and it would hopefully catch it, `isDisabled` strongly suggests to readers it is a boolean.

Reply:
Good call, I will change that 👍